src/pydvl/value/oob/oob.py

```diff
@@ -0,0 +1,139 @@
+"""
+## References
+
+[^1]: <a name="kwon_data_2023"></a>Kwon et al. [Data-OOB: Out-of-bag Estimate as a Simple and Efficient Data Value]
+(https://proceedings.mlr.press/v202/kwon23e.html). In: Published at ICML 2023
```

Comment:
For markdown links to be rendered properly there shouldn't a new line in the middle.

```suggestion
[^1]: <a name="kwon_data_2023"></a>Kwon et al.
[Data-OOB: Out-of-bag Estimate as a Simple and Efficient Data Value](https://proceedings.mlr.press/v202/kwon23e.html).
In: Published at ICML 2023
```